src/wallet/rpcwallet.cpp

```diff
@@ -4361,12 +4361,13 @@ UniValue z_viewtransaction(const UniValue& params, bool fHelp)
         auto op = res->second;
         auto wtxPrev = pwalletMain->mapWallet.at(op.hash);
 
-        // We don't need to check the leadbyte here: if wtx exists in
+        // We don't need to constrain the plaintext leadbyte to
+        // satisfy the ZIP 212 grace window: if wtx exists in
         // the wallet, it must have been successfully decrypted. This
         // means the plaintext leadbyte was valid at the block height
         // where the note was received.
         // https://zips.z.cash/zip-0212#changes-to-the-process-of-receiving-sapling-notes
```

Comment:
```suggestion
        // https://zips.z.cash/zip-0212#changes-to-the-process-of-receiving-sapling-or-orchard-notes
```
(the given link doesn't exist).